utility_analysis_new/combiners.py

```diff
@@ -569,14 +546,27 @@ def compute_metrics(self,
 
     def metrics_names(self) -> List[str]:
         return [
-            'abs_error_expected', 'abs_error_variance', 'abs_error_quantiles',
-            'rel_error_expected', 'rel_error_variance', 'rel_error_quantiles'
+            'metric_type', 'abs_error_expected', 'abs_error_variance',
+            'abs_error_quantiles', 'rel_error_expected', 'rel_error_variance',
+            'rel_error_quantiles'
         ]
 
     def explain_computation(self):
         pass
 
 
+class PrivacyIdCountAggregateErrorMetricsCombiner(
+        CountAggregateErrorMetricsCombiner):
```

Comment:
I thought again. 

WDYT about avoiding class inheritance here, i.e. use `CountAggregateErrorMetricsCombiner`, and to have 
`metrics.AggregateMetricType` in the constructor of this class. It seems simpler, no additional classes.